python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -40,6 +40,14 @@
 from tests.util.date import DateRange
 
 
+if sys.platform == "win32":
+    # leave out Mongo as spinning up a Mongo instance in Windows CI is fiddly, and Mongo support is only
```

Comment:
Curious. I thought the Windows runner already had Mongo service installed. You just need to add a command to start it